getting-started/setting-env.md

```diff
@@ -76,6 +76,7 @@ Also for this tutorial we will use these variables. So make sure you export thes
 
 ```bash
 export NODE=(--node "https://rpc.musselnet.cosmwasm.com:443")
+export CHAIN_ID=musselnet-4
```

Comment:
This is the only change still specific to musselnet-4, I think we should define the $CHAIN_ID variable if we use it in the next line.